packages/fstar/fstar.0.9.0/files/fstar.install

```diff
@@ -0,0 +1,2 @@
+bin: [ "?bin/fstar" { "fstar" } ]
+bin: [ "?bin/fstar.exe" { "fstar.exe" } ]
```

Comment:
I.e. do not repeat the `bin`.  Since I suppose you read [the manual](https://opam.ocaml.org/doc/Manual.html#packagenameinstall), here is [an example](https://github.com/ocaml/opam-repository/blob/master/packages/oasis2opam/oasis2opam.0.6.2/files/oasis2opam.install).